migrations/20200702154715-bulk_entity_mapping_sample_csv_upload.js

```diff
@@ -0,0 +1,68 @@
+
+const fs = require('fs');
+
+const https = require('https');
+const request = require('request');
+
+module.exports = {
```

Comment:
@rakeshSgr - Same changes as mentioned in other migration files of sample file upload